miniwallet/deploy/001_deploy_miniWallet.ts

```diff
@@ -0,0 +1,85 @@
+import config from '../config'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { DeployFunction } from 'hardhat-deploy/types'
+import { ethers } from 'hardhat'
+
+const OPERATOR_ROLE = ethers.utils.id('OPERATOR_ROLE')
+
+const deployFunction: DeployFunction = async function (
+  hre: HardhatRuntimeEnvironment
+) {
+  const { deployments, getNamedAccounts, getChainId } = hre
+  const { deploy } = deployments
+  const { deployer } = await getNamedAccounts()
+  const chainId = await getChainId()
+  let initialOperatorThreshold
+  let initialOperators
+  let initialUserLimit
+  let initialAuthLimit
+
+  console.log(`chainId: ${chainId}`)
+  if (chainId === '1666600000,') {
```

Comment:
extra ,

Reply:
Refactored approach and commit are [here](https://github.com/polymorpher/sms-wallet/issues/13#issuecomment-1246191757)